extensions/amp-story-shopping/0.1/test/test-amp-story-shopping-config.js

```diff
@@ -229,5 +261,145 @@ describes.realWin(
         ).to.have.been.calledOnce;
       });
     });
+
+    it('test validate required fields for config', async () => {
+      const invalidConfig = {
+        'items': [{}],
+      };
+
+      const validateRequiredSpy = env.sandbox.spy(
+        AmpStoryShoppingConfig,
+        'validateRequired'
+      );
+
+      await createAmpStoryShoppingConfig(null, invalidConfig);
+
+      for (const [key, value] of Object.entries(
+        VALIDATION_OBJECTS['productValidationConfig']
+      )) {
+        if (value.includes(validateRequiredSpy)) {
+          expect(() => {
+            validateRequiredSpy(key, invalidConfig['items'][0][key]);
```

Comment:
This test is bad: you call the public API and don't use it in any way (you could literally remove it and your test would still pass). You then call a private method manually that has a very very low test coverage to write your `expect`